### DISCUSSION TITLE
I can't find a way to imporve my model's performance

### DISCUSSION THREAD

**Discussion by purple-mustache** [mrdbourke/pytorch-deep-learning#830] [Q&A]
Initially, the model was overfitting, so I added a dropout layer, but now it keeps underfitting. I have tried increasing the learning rate and training for longer to get the performance up, but the current performance is still worse than random guessing on both the train and test data. Any ideas for how I can overcome this? (I'm using the pizza_sushi_steak data)

**Comment by LuluW8071** [comment]
Well maximum i could get was 60% on test accuracy. Though, it is understandable that your model is undergoing either **overfitting** or **underfitting**. It could be mainly due to 
- Less datasets: The video approach had only 20% of dataset taken into account
- Less Image Reshape Size: The video approach took `64x64` px of image size try taking `224x224` px of image size
- Add BatchNorm() technique
- Add more conv_block layers 

In my case I took 30% data with addding more conv_block layers
Note: I just trained fewer epochs in my case so results may vary if u train on more epochs
```
Epoch: 1 | Train loss: 0.9268 - Train acc: 59.79% -  Test loss: 0.8194 - Test acc: 62.08%
Epoch: 2 | Train loss: 0.9067 - Train acc: 61.43% -  Test loss: 0.8173 - Test acc: 61.67%
Epoch: 3 | Train loss: 0.9262 - Train acc: 60.27% -  Test loss: 0.8348 - Test acc: 60.83%
Epoch: 4 | Train loss: 0.9582 - Train acc: 55.47% -  Test loss: 0.8261 - Test acc: 60.42%
Epoch: 5 | Train loss: 0.9567 - Train acc: 58.62% -  Test loss: 0.8232 - Test acc: 62.08%
```
